README.md

```
@@ -1,41 +1,427 @@
+<img src="https://helixque.vercel.app/logo.svg" alt="Helixque Logo" width="200" />
 
-First, run the development server:
+
+> **Official landing page for [Helixque](https://helixque.vercel.app/)** - a professional networking platform where developers, designers, founders, and learners connect for instant matching based on skills, industry, and language.
+### Helixque Landing Page
+
+</div>
+
+## Table of Contents
+
+- [About Helixque](#-about-helixque)
```

markdownlint-cli2 (0.18.1)

Link fragments should be valid

(MD051, link-fragments)